spaceship/lib/spaceship/connect_api/models/build.rb

```diff
@@ -35,6 +36,56 @@ class Build
       def self.type
         return "builds"
       end
+
+      #
+      # Helpers
+      #
+
+      def app_version
+        raise "No pre_release_version included" unless pre_release_version
+        return pre_release_version.version
+      end
+
+      def app_id
+        raise "No app included" unless app
+        return app.id
+      end
+
+      def bundle_id
+        raise "No app included" unless app
+        return app.bundle_id
+      end
+
+      def processed?
+        return processing_state == "VALID"
+      end
+
+      # This is here temporarily until the removal of Spaceship::TestFlight
+      def to_testflight_build
+        h = {
+          'buildVersion' => version,
+          'uploadDate' => uploaded_date,
+          'externalState' => processed? ? Spaceship::TestFlight::Build::BUILD_STATES[:active] : Spaceship::TestFlight::Build::BUILD_STATES[:processing],
+          'appAdamId' => app_id,
+          'bundleId' => bundle_id,
+          'trainVersion' => app_version
+        }
+
+        return Spaceship::TestFlight::Build.new(h)
+      end
+
+      #
+      # API
+      #
+
+      def self.all(app_id: nil, version: nil, build_number: nil, includes: nil)
+        resps = client.get_builds(
+          filter: { app: app_id, "preReleaseVersion.version" => version, version: build_number },
+          includes: includes,
+          limit: 30
+        ).all_pages
+        return resps.map(&:models).flatten
```

Comment:
Is the `flatten` called to turn the pages-of-lists to a 1D list?

Reply:
Yup yup, that is correct 👌